fgpyo/samtools.py

```diff
@@ -0,0 +1,375 @@
+"""
```

Comment:
Does this file automatically show up in generated docs?

I know we'll eventually need to fixup the VCF work so it is indexed: https://github.com/fulcrumgenomics/fgpyo/issues/72